pkg/operator/operator.go

```diff
@@ -138,17 +138,36 @@ func (optr *Operator) Run(workers int, stopCh <-chan struct{}) {
 	defer utilruntime.HandleCrash()
 	defer optr.queue.ShutDown()
 
-	glog.Info("Starting MachineAPIOperator")
+	glog.Info("Starting MachineAPIO--rator")
 	defer glog.Info("Shutting down MachineAPIOperator")
 
+	// TODO(alberto) operatorConfig as CRD?
```

Comment:
how about we run this in sync function

```
	if err := optr.syncClusterAPIServer(*operatorConfig); err != nil {
		return err
	}
	if err := optr.syncClusterAPIController(*operatorConfig); err != nil {
		return err
	}
       return optr.syncAll(*operatorConfig)
```
then `syncAll()` syncs machinesets/cluster